### ISSUE TITLE
Migrate to Python 3.10

### ISSUE THREAD

**Issue by bennahugo** [ratt-ru/montblanc#280]
Already started fixing this. TF 2.8.4 works, so in principle we should be able to run on 3.10. Later versions of Tensorflow gives an obscure symbol issue which I haven't tracked down yet.

**Comment by bennahugo** [comment]
Also being blocked by Python 3.10 issues on Meqtrees Timba, which needs resolving. I will open a separate ticket for that soon
